Automation_Lesson1/Task4/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace Task4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string stringInput1 = "";
+            string stringInput2 = "";
+            string symbol;
+
+            Console.WriteLine("Input first part of the string: ");
+            stringInput1 = Console.ReadLine();
+            while (stringInput1 == "")
+            {
+                Console.WriteLine("Input non-empty first part of the string:");
+                stringInput1 = Console.ReadLine();
+            }
+
+            Console.WriteLine("Input second part of the string: ");
+            stringInput2 = Console.ReadLine();
+            while (stringInput2 == "")
+            {
+                Console.WriteLine("Input non-empty second part of the string:");
+                stringInput2 = Console.ReadLine();
+            }
+
+            string stringConcatenated = stringInput1 + stringInput2;
+
+            Console.WriteLine("Input symbol: ");
```

Comment:
@twinkleyarns to input the char, it's easier to use another Console function, without the strange checks, pleasy try to found it and use